pkgs/development/libraries/zlib-ng/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib, stdenv, fetchFromGitHub
+, cmake, pkg-config
+, withZlibCompat ? false
```

Comment:
Would we want to enable this by default to make it a drop in?

Reply:
Disabled in upstream by default. This option is needed if you completely replace the zlib library. Updated PR.